src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.ArrayList;
+
+
+/**
+ * Lists all persons in the address book to the user in an order sorted by name alphabetically ignoring case.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all persons in the address book as a list with index numbers in an order sorted by name alphabetically ignoring case.\n"
+            + "Example: " + COMMAND_WORD;
+
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.